tests/testutil.c

```diff
@@ -408,7 +408,7 @@ test_util_get_system_module_name (void)
     char * result;
 
     _test_util_system_module_name = malloc (64);
-    p = popen ("find /lib -name \"libc-*.so\" | head -1 | xargs basename"
+    p = popen ("find /lib -name \"libc[-.]*so*\" | head -1 | xargs basename"
```

Comment:
How about running the old/current command first, and if that yields nothing we try your glob second?